crates/yttrium/src/transaction/send/simple_account_test.rs

```diff
@@ -238,23 +238,26 @@ mod tests {
 
         println!("Received User Operation hash: {:?}", user_operation_hash);
 
-        // let receipt = bundler_client
-        //     .get_user_operation_receipt(user_operation_hash.clone())
-        //     .await?;
+        // TODO convert to polling
+        tokio::time::sleep(Duration::from_secs(2)).await;
 
-        // println!("Received User Operation receipt: {:?}", receipt);
+        let receipt = bundler_client
+            .get_user_operation_receipt(user_operation_hash.clone())
+            .await?;
+
+        println!("Received User Operation receipt: {:?}", receipt);
 
-        // println!("Querying for receipts...");
+        println!("Querying for receipts...");
 
-        // let receipt = bundler_client
-        //     .wait_for_user_operation_receipt(user_operation_hash.clone())
-        //     .await?;
+        let receipt = bundler_client
+            .wait_for_user_operation_receipt(user_operation_hash.clone())
```

Comment:
Last time I tried to run this, I was getting an error, which is why it was commented out, is this now working?

Reply:
Yeah it's working now